cerberus-webapp/src/main/java/ca/on/oicr/gsi/cerberus/ProvenanceServlet.java

```diff
@@ -30,7 +30,7 @@
  *
  * @author ibancarz
  */
-@WebServlet(name = "ProvenanceServlet")
+@WebServlet(name = "ProvenanceServlet", urlPatterns = { "/provenance" })
```

Comment:
Is this a reversion?

Reply:
@iainrb what other servlets would be on that machine that would conflict with this one?

Reply:
@apmasell Yes.
@callunity StatusServlet.java has:
`urlPatterns = { "/" }`
Which I believe is a conflict. At least it stopped the CLI from working with an installation on OpenStack.

Reply:
Why is having the suffix a problem?

Reply:
Yeah. I think that the requirements of that ticket have changed.

Reply:
Okay, I'll make a note on the ticket, and make the other changes you suggested above.

Reply:
Latest commit hopefully includes all needed changes, let me know what you think.